game-backend/src/game-engine/intent/MoveIntent.test.ts

```diff
@@ -1,26 +1,63 @@
 import MoveIntent, { Direction } from './MoveIntent';
+import StateBuilder from '../../test-helper/StateBuilder';
+import { ARENA_HEIGHT } from '../../../../darwin-types/Arena';
 
 describe('MoveIntent', () => {
   it('handles move properly', () => {
     const intent = new MoveIntent(Direction.Down);
 
     const newState = intent.execute(
-      {
-        objectMap: {
-          unit1: {
-            id: 'unit1',
-            position: {
-              x: 0,
-              y: 0,
-            },
-          },
-        },
-        objectIds: ['unit1'],
-      },
+      StateBuilder.buildState()
+        .addObject({ id: 'unit1', x: 0, y: 0 })
+        .build(),
       {
         unitId: 'unit1',
       }
     );
     expect(newState.objectMap.unit1.position.y).toBe(1);
```

Comment:
Where do we want the origin of our coordinate system to be in? in the center, top-left, bottom-left?
Imo it should be in the center or bottom-left. Having a "move down" intent increasing a axis value feels really weird.
What do you guys think? @sir-marc @flipmess @ferinator @mailaenderli @StuckiSimon

Reply:
For rendering in the frontend the origin will be top left (`canvas`). Therefore I'd like to use the same reference point in the backend as well.

Reply:
That's a good point. However, I still find the Cartesian coordinate system way more intuitive. The canvas is also not doing any calculations with the coordinates. Flipping the y-axis should happen in the frontend and not in the backend imo. 

But maybe it's just me, who feels so put off by a y-axis pointing to the right of the x-axis.

Reply:
As discussed with the team, we'll use the upper left as the origin of the coordinate system.